service/src/registry.ts

```diff
@@ -307,7 +314,13 @@ export class Registry {
   /**
    * @method            module:azure-iothub.Registry#updateDevices
    * @description       Updates an array of devices.
-   *
+   *                    The individual elements of the `devices` parameter must
+   *                    include all properties that are updatable.  For example,
+   *                    if updating the `status` property from `disabled` to
+   *                    `enabled`, the object should also have the statusReason,
+   *                    authentication (and its sub-properties), capabilities,
+   *                    and deviceScope set to the devices current values or those
+   *                    properties will be reset.
```

Comment:
Thoughts on just including a link to the IoT Hub documentation as well so they know what properties are "updatable" ? [Device Identity properties](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#device-identity-properties)

Reply:
also, that documentation page doesn't necessarily describe what is "updatable"... do you know this? I'd like to add a column to that page to make this clear

Reply:
It would be good to add a column to the doc page.  I am hesitant to add a link.  It could easily become invalid.